### PR TITLE
Update supported functions from morphir-scala

### PR DESCRIPTION
**Pull request by bensanchin**
Updated supported functions from [morphir-scala](https://github.com/finos/morphir-scala).

### CODE REVIEW

**Review by michelchan — APPROVED** [review]
lgtm. thanks for updating this

File: docs/scala/support.md [hunk 1]
```diff
@@ -157,56 +157,56 @@ This section lists the SDK types and functions currently supported in this versi
...
+| toLower       | v2.31.3                        | &#x2713;  | 
+| toLocaleUpper | v2.31.3                        | &#x2713;  | 
+| toLocaleLower | v2.31.3                        | &#x2713;  | 
+| toCode        | v2.31.3                        | &#x2713;  | 
+| fromCode      | v2.31.3                        | &#x2713;  | 
 
 ### Morphir.SDK.Decimal
 | Function          | Morphir-Elm Version Introduced | Supported |
 |-------------------|--------------------------------|-----------|
-| fromInt           | v2.8.0                         | &#x2713;  |
```
Comment (bensanchin) [inline docs/scala/support.md:177]: Any concerns if we sort the table? Thought it be easier on eyes if the list is sorted.

Comment (michelchan) [reply docs/scala/support.md:177]: It's currently in the order that the IR functions are listed. Personally I don't think there's any need to sort it

Comment (bensanchin) [reply docs/scala/support.md:177]: @michelchan Thanks for the clarification. I will revert the sort change.

Comment (bensanchin) [reply docs/scala/support.md]: Reverted the result from sort.

File: docs/scala/support.md [hunk 2]
```diff
@@ -423,8 +423,8 @@ This section lists the SDK types and functions currently supported in this versi
 | addHours           | v2.44.0                        | &#x2713;  |
 | addMinutes         | v2.44.0                        | &#x2713;  |
 | addSeconds         | v2.44.0                        | &#x2713;  |
-| diffInHours        | v2.44.0                        | &#x2715;  |
-| diffInMinutes      | v2.44.0                        | &#x2715;  | 
+| diffInHours        | v2.44.0                        | &#x2713;  |
```
Comment (bensanchin) [inline docs/scala/support.md:426]: See https://github.com/finos/morphir-scala/blob/e5f54adf5b400005e6591aa6d80f3d45305cb6d9/morphir/src/org/finos/morphir/runtime/sdk/LocalTimeSDK.scala#L51

File: docs/scala/support.md [hunk 3]
```diff
@@ -434,12 +434,11 @@ This section lists the SDK types and functions currently supported in this versi
 |-------------|--------------------------------|-----------|
 | withDefault | v2.84.2                        | &#x2713;  |
 | map         | v2.84.2                        | &#x2713;  |
-| map2        | v2.84.2                        | &#x2715;  |
-| map3        | v2.84.2                        | &#x2715;  |
-| map4        | v2.84.2                        | &#x2715;  |
-| map4        | v2.84.2                        | &#x2715;  | 
+| map2        | v2.84.2                        | &#x2713;  |
```
Comment (bensanchin) [inline docs/scala/support.md:437]: See https://github.com/finos/morphir-scala/blob/e5f54adf5b400005e6591aa6d80f3d45305cb6d9/morphir/src/org/finos/morphir/runtime/sdk/MaybeSDK.scala#L99

**Pull request merged**